ios/Podfile.lock

```diff
@@ -254,13 +254,13 @@ PODS:
     - React-cxxreact (= 0.63.2)
     - React-jsi (= 0.63.2)
   - React-jsinspector (0.63.2)
-  - react-native-exposure-notification-service (1.1.28):
+  - react-native-exposure-notification-service (1.1.30):
```

Comment:
Do we need to bump this in the package-json too?

Reply:
```
   "react-native-exposure-notification-service": "^1.1.28",
```

Reply:
No need i think, as yarn lock is up to date and so is the Podfile.lock (actually this is the main reason I did this update).